app/scripts/modules/core/src/presentation/HoverablePopover.tsx

```diff
@@ -165,6 +165,7 @@ export class HoverablePopover extends React.Component<IHoverablePopoverProps, IH
           placement={placementOverride || placement}
           target={this.targetRef.current}
           container={container}
+          shouldUpdatePosition={true}
```

Comment:
Adding this because we're actually changing the children in the popover (instead of creating an entirely new one), which means we need to recompute its position when children change.

I only 30% understand why this works.